tests/appsec/iast/sink/test_xcontent_sniffing.py

```diff
@@ -0,0 +1,50 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import context, coverage, released, missing_feature
+from .._test_iast_fixtures import SinkFixture
+
+
+@coverage.basic
+@released(dotnet="?", golang="?", php_appsec="?", python="?", ruby="?", nodejs="?")
+@released(java={"akka-http": "?", "ratpack": "?", "spring-boot-3-native": "?", "*": "1.18.0"})
```

Comment:
Could you use the manifest files to declare those version please ?

Reply:
And apprt from that, the CI is failing on this new test.

Reply:
Applied